api/src/passports/single-use-code.strategy.ts

```diff
@@ -87,6 +87,7 @@ export class SingleUseCodeStrategy extends PassportStrategy(
       },
     });
     if (!rawUser) {
+      // this is failing
```

Comment:
Have struggled w the most secure thing to do here @YazeedLoonat

Reply:
maybe we sync up on this, but I think whats happening is:

- user has account with email
- they go to update their email to new email
- we send an email to their new email with a code, but we don't immediately update their user to have the new email. Instead we save the new email in the confirmation token in the db and once they successfully log in we do the email update
- the front end is sending the new email + code to the backend to be logged in. But that new email doesn't exist yet

I think the front end needs to send the original email + code and then the new email will take

hopefully that helps!